src/DOMParser.ts

```diff
@@ -163,6 +163,20 @@ export class DOMParser {
       current.appendChild(doc.createComment(comment));
     };
 
+    //XML needs to start with <...(root element) or <?...(prolog) but not a comment <!-...
+    if (
+      source != null &&
+      (source.startsWith('<') !== true || source.startsWith('<!-') === true)
+    ) {
+      errors.push(
+        new XMLError(
+          'XML must start with prolong or root element.',
```

Comment:
should be "prolog"

Reply:
Fixed in [51cdae6](https://github.com/journeyapps/domparser.js/pull/22/commits/51cdae65e645a59c42696f798cb6337f085921e2), Unit test has also been added.